binding/springjms/src/main/java/io/tracee/binding/springjms/TraceeMessageConverter.java

```diff
@@ -0,0 +1,54 @@
+package io.tracee.binding.springjms;
+
+import io.tracee.Tracee;
+import io.tracee.TraceeBackend;
+import org.springframework.jms.support.converter.MessageConversionException;
+import org.springframework.jms.support.converter.MessageConverter;
+import org.springframework.jms.support.converter.SimpleMessageConverter;
+
+import javax.jms.JMSException;
+import javax.jms.Message;
+import javax.jms.Session;
+
+public class TraceeMessageConverter implements MessageConverter {
+
+	private final MessageConverter delegate;
+
+	private final TraceeBackend backend;
+
+	public TraceeMessageConverter() {
+		this.delegate = new SimpleMessageConverter();
+		this.backend = Tracee.getBackend();
+	}
+
+	public TraceeMessageConverter(MessageConverter delegate) {
+		this.delegate = delegate;
+		this.backend = Tracee.getBackend();
+	}
+
+	TraceeMessageConverter(MessageConverter converter, TraceeBackend backend) {
+		this.delegate = converter;
+		this.backend = backend;
+	}
+
+	@Override
+	public Message toMessage(Object o, Session session) throws JMSException, MessageConversionException {
+		final Message message = delegate.toMessage(o, session);
```

Comment:
What if the delegate fails (JMSException, MessageConversionException)? Should we still try to write the context to the message?